docs/upgrade/v1-1-2-to-v1-2-0.md

```diff
@@ -483,3 +484,93 @@ After performing the steps above, you should pass post-draining with the next re
+    harvester-system            hvst-upgrade-vq4hl-apply-manifests-65vv8                   1/1     Running     0              87m  // waiting for managedchart to be ready
+    ..
+    ```
+
+2. The pod `hvst-upgrade-vq4hl-apply-manifests-65vv8` has the following loop log:
+
+    ```
+    Current version: 102.0.0+up40.1.2, Current state: WaitApplied, Current generation: 23
+    Sleep for 5 seconds to retry
+    ```
+
+3. Check the status for all bundles. Note thata couple of bundles are `OutOfSync`:
+
+    ```
+    # kubectl get bundle -A
+    NAMESPACE     NAME                                          BUNDLEDEPLOYMENTS-READY   STATUS
+    ...
+    fleet-local   mcc-local-managed-system-upgrade-controller   1/1
+    fleet-local   mcc-rancher-logging                           0/1                       OutOfSync(1) [Cluster fleet-local/local]
+    fleet-local   mcc-rancher-logging-crd                       0/1                       OutOfSync(1) [Cluster fleet-local/local]
+    fleet-local   mcc-rancher-monitoring                        0/1                       OutOfSync(1) [Cluster fleet-local/local]
+    fleet-local   mcc-rancher-monitoring-crd                    0/1                       WaitApplied(1) [Cluster fleet-local/local]
+    ```
+
+4. The pod `fleet-agent-*` has following error log:
+
+    ```
+    fleet-agent POD log:
+
+    time="2023-09-19T12:18:10Z" level=error msg="Failed to register agent: looking up secret cattle-fleet-local-system/fleet-agent-bootstrap: Post \"https://192.168.122.199/apis/fleet.cattle.io/    v1alpha1/namespaces/fleet-local/clusterregistrations\": tls: failed to verify certificate: x509: cannot validate certificate for 192.168.122.199 because it doesn't contain any IP SANs"
+    ```
+
+5. Check the `ssl-certificates` settings in Harvester:
+
+From the command line:
+
+    ```
+    # kubectl get settings.harvesterhci.io ssl-certificates
+    NAME               VALUE
+    ssl-certificates   {"publicCertificate":"-----BEGIN CERTIFICATE-----\nMIIFNDCCAxygAwIBAgIUS7DoHthR/IR30+H/P0pv6HlfOZUwDQYJKoZIhvcNAQEL\nBQAwFjEUMBIGA1UEAwwLZXhhbXBsZS5j...."}
+    ```
+
+From the Harvester Web UI:
+
+![](/img/v1.2/upgrade/known_issues/4519-harvester-settings-ssl-certificates.png)
+
+6. Check the `server-url` setting, it is the value of VIP:
+
+    ```
+     # kubectl get settings.management.cattle.io -n cattle-system server-url
+    NAME         VALUE
+    server-url   https://192.168.122.199
+    ```
+
+7. The root cause of this issue is when you set self-signed `ssl-certificates` with FQDN in the Harvester settings, and the `server-url` points to the VIP.
+
+    ```
+    e.g. create certificate for (*).example.com
+
+    openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes \
+    -keyout example.key -out example.crt -subj "/CN=example.com" \
+    -addext "subjectAltName=DNS:example.com,DNS:*.example.com"
+
+    the general outputs are: example.crt  example.key
+    ```
```

Comment:
From the comment, It looks like the [workaround](https://github.com/harvester/harvester/issues/4519#issuecomment-1727132383) action is to do:
```
(7) update `server-url` with the value of `https://harv31.example.com`
 # kubectl edit settings.management.cattle.io -n cattle-system server-url
setting.management.cattle.io/server-url edited
```

Should we also include it there?